package.json

```diff
@@ -7,7 +7,7 @@
   "scripts": {
     "test:deps": "dependency-check . && dependency-check . --extra --no-dev -i es2040",
     "test:lint": "standard",
-    "test:node": "NODE_ENV=test bin/test",
+    "test:node": "NODE_ENV=test tape **/*.test.js",
```

Comment:
did [`bin/test`](https://github.com/ahdinosaur/vas/blob/v3/bin/test) not do this already? the reason i added this script is so you can also use args to run specific tests like `npm run test:node -- index.test.js`. should really be a module though... :stuck_out_tongue_closed_eyes: